docs/gettingstarted/index.rst

```diff
@@ -162,11 +162,9 @@ Getting started with Flocker
 
          Simulate a Flocker cluster with virtual machines on your laptop (requires `Vagrant <http://www.vagrantup.com/downloads>`_, `VirtualBox <https://www.virtualbox.org/wiki/Downloads>`_):
 
-         .. code-block:: console
+         .. version-code-block:: console
 
-            you@laptop:~$ git clone \
-              https://github.com/clusterhq/vagrant-flocker && \
-              cd vagrant-flocker && \
+            you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/Vagrantfile && \
```

Comment:
The old approach gave you a directory dedicated to the vagrant stuff being created.  That might have been a nice thing.  If you agree, I suggest inserting `mkdir vagrant-flocker && cd vagrant-flocker && ...` at the front of this and the following command.